core/changedetector/change_detector.go

```diff
@@ -0,0 +1,24 @@
+package changedetector
+
+import "time"
+
+type ChangeRequest struct {
+	NamespaceID string
+	SchemaName  string
```

Comment:
this is different. Since, create Schema does not return schemaID, hence passing name to get schema Id.
The one you are referring is NotificationEvent